src/box-client.ts

```diff
@@ -13,19 +12,22 @@
  * A collaboration role constant
  * @typedef {string} CollaborationRole
  */
+type CollaborationRole = string;
 
 /**
-* A Box file or folder type constant
-* @typedef {string} ItemType
-*/
+ * A Box file or folder type constant
+ * @typedef {string} ItemType
+ */
+type ItemType = string;
 
 /**
  * An access level constant. Used for setting and updating shared links, folder upload, etc.
  * @typedef {?Object} AccessLevel
  */
+type AccessLevel = Object | null;
```

Comment:
Should we add a `/* FIXME */` note to update this type to be more specific later on? Or will it remain defined as `Object`?

Reply:
Also - still learning more the nuances of typescript myself, but it looks like we want lowercase `object` here instead of uppercase `Object`?

- https://www.typescriptlang.org/docs/handbook/basic-types.html#about-number-string-boolean-symbol-and-object
- https://2ality.com/2020/01/typing-objects-typescript.html#object-vs.-object%3A-primitive-values

Reply:
yes, good point